### PR TITLE
Implement Java Math functions in ballerina

### PR DESCRIPTION
**Pull request by arukshani**
Note: This needs to be merged after the language changes.

### CODE REVIEW

File: modules/ballerina-native/src/main/ballerina/ballerina/lang/math/natives.bal [hunk 1]
```diff
@@ -28,3 +28,275 @@ native function randomInRange (int start, int end) (int);
...
+@doc:Param { value:"val: an angle, in degrees" }
+@doc:Return { value:"float: the measurement of the angle angdeg in radians" }
+native function toRadians (float val) (float);
+
+@doc:Description { value:"Returns the size of an ulp of the argument"}
+@doc:Param { value:"val: the floating-point value whose ulp is to be returned" }
+@doc:Return { value:"float: the size of an ulp of the argument" }
+native function ulp (float val) (float);
+
+
```
Comment (hasithaa) [inline modules/ballerina-native/src/main/ballerina/ballerina/lang/math/natives.bal]: Please remove extra white space.

**Comment by arukshani** [comment]
It's already there Hashitha. Please check "MathTest.java".

Here's the link >> https://github.com/ballerinalang/ballerina/pull/3299/files#diff-3fb9c2f899337a027825a699b99cd61c

**Comment by arukshani** [comment]
Please ignore this pull request and consider the following pull request instead. 

https://github.com/ballerinalang/ballerina/pull/3667

**Comment by hasithaa** [comment]
Closing this PR, as requested.

**Pull request closed without merging**